languages/typescript/src/main/kotlin/io/vrap/codegen/languages/typescript/joi/JoiValidatorModuleRenderer.kt

```diff
@@ -127,12 +128,11 @@ class JoiValidatorModuleRenderer @Inject constructor(override val vrapTypeProvid
         }
     }
 
-    fun Property.renderPropertyTypeSchema(): String {
-        val owner: ObjectType = eContainer() as ObjectType
+    fun Property.renderPropertyTypeSchema(current : ObjectType): String {
```

Comment:
Here we need to pass in for which property we render. This is because we render all properties and they have different owners.